packages/back-end/test/services/settings/settings.test.ts

```diff
@@ -130,13 +136,16 @@ describe("settings", () => {
           expect(metricSettings_testvar.conversionWindowHours.value).toEqual(
             72
           );
-          expect(
-            metricSettings_testvar.regressionAdjustmentEnabled.value
-          ).toEqual(false);
-          expect(
-            metricSettings_testvar.regressionAdjustmentEnabled.meta.reason ===
-              "custom aggregation"
-          );
+          // Q for Bryce - Does metric_testvar need  to be of type 'binomial' for custom aggregation override to work?
```

Comment:
ugh good question. We have different treatment here for frontend versus backend services (this is a bug). I believe the only thing that matters is `if(aggregation)`, and the "binomial" check is leftover from an earlier conversation.

Reply:
Updated to ignore metric type